pkg/providers/aws/provider_redis.go

```diff
@@ -137,11 +135,11 @@ func (p *RedisProvider) CreateRedis(ctx context.Context, r *v1alpha1.Redis) (*pr
 		return nil, croType.StatusMessage(errMsg), errorUtil.Wrap(err, errMsg)
 	}
 
-	//networkManager isEnabled checks for the presence of valid RHMI subnets in the cluster vpc
-	//when rhmi subnets are present in a cluster vpc it indicates that the vpc configuration
+	//networkManager isEnabled checks for the presence of valid CRO subnets in the cluster vpc
```

Comment:
@cecobask , sorry, I didn't look at AWS yet, just wanted to ask if it's RHOAM subnets or CRO ? (as before it was RHMI subnets)

Reply:
@valerymo These are subnets created by CRO, regardless if used by RHOAM or CRO on its own.